LFS171x/fabric-material/tuna-app/.gitignore

```diff
@@ -0,0 +1,25 @@
+# Emacs backup files
+*~
+*#
+.#*
+# Vim file artifacts
+.*.sw*
+# installed platform-specific binaries
+/bin
+.DS_Store
+.project
+
+first-network/channel-artifacts/*.tx
+first-network/channel-artifacts/*.block
+first-network/crypto-config/*
+
+chaincode-docker-devmode/myc.block
+chaincode-docker-devmode/chaincode/sacc/sacc
+chaincode-docker-devmode/chaincode/chaincode_example02/chaincode_example02
+
+# fabric sdk node modules
+tuna-app/node_modules/
+
+# balance transfer sample
```

Comment:
We don't have a balance transfer sample.

Reply:
removed